src/chapters/08-working-with-data.md

```diff
@@ -2,9 +2,24 @@
 ember server
 ```
 
+In this chapter, we will remove the hard-coded data from our `<Rental>` component. By the end, your app should look the same, but with the ability to `fetch` this data from an API.
```

Comment:
```suggestion
In this chapter, we will remove the hard-coded data from our `<Rental>` component. By the end, your app would finally be displaying real data that came from the server!
```

It doesn't actually look the same, because the single duplicated listing will now be replaced with three "real" rental listing.

Reply:
```suggestion
In this chapter, we will remove the hard-coded data from our `<Rental>` component. By the end, your app will finally display real data that will come from the server!
```

Reply:
"In this chapter, we will remove the hard-coded data from our `<Rental>` component. By the end, your app will finally display real data that comes from the server!"